bundles/org.openhab.binding.semsportal/README.md

```diff
@@ -0,0 +1,45 @@
+
+## Discovery
+
+At this moment there is no discovery possible. Configuration of the powerStation unique id is done by hand in the Thing configuration page.
+
+## Thing Configuration
+
+The configuration of the thing is pretty straight forward. You need to have your power station set up in the SEMS portal, and you need to have an account that is allowed to view the power station data. You should log in at least once in the portal with this account to activate it. 
+
+The thing needs the username and password to connect and retreive the data. It also needs the unique id of the power station. It can be found after you log in to the portal, on the URL. use the value you find in the URL at the x-es (including the -s) in this example: https://www.semsportal.com/powerstation/powerstatussnmin/xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+
+
+## Channels
+
+
+| channel       | type      | description                                                                                                |
+| ------------- | --------- | ---------------------------------------------------------------------------------------------------------- |
+| onlineState   | Switch    | If the powerStation is currently on and reporting to the portal                                            |
+| lastUpdate    | DateTime  | Last time the powerStation sent information to the portal                                                  |
+| currentOutput | Number    | The current output of the powerStation in Watt                                                             |
+| todayTotal    | Number    | Todays total generation of the station in kWh                                                              |
+| monthTotal    | Number    | This month's total generation of the station in kWh                                                        |
+| overallTotal  | Number    | The total generation of the station since installation, in kWh                                             |
+| todayIncome   | Number    | Todays income as reported by the portal, if you have configured the power rates of your energy provider    |
+| totalIncome   | Number    | The total income as reported by the portal, if you have configured the power rates of your energy provider |
+
+## Parameters
+
+| parameter   | required? | description                                                                                                |
+| ----------- |:---------:| ---------------------------------------------------------------------------------------------------------- |
+| username    | X         | Account name (emailaddress) at the SEMS portal. Account must have been used at least once to log in.       |
```

Comment:
```suggestion
| username    | X         | Account name (email address) at the SEMS portal. Account must have been used at least once to log in.       |
```

No need to separate these onto new lines for each sentence when they are in a table/grid format.